src/boost/formatters.nim

```diff
@@ -64,21 +72,16 @@ proc alignStr*(s: string, len: int, fill = ' ', trunc = false): string =
   if len == 0:
     result = s
   elif trunc and s.len > abs(len):
+    result = s
```

Comment:
This shouldn't matter anyway if we're to disable default truncation. Then the behaviour would only be relevant for direct uses of `alignStr`.

In my opinion, `alignStr` tries to do a lot of things at once, so it gets too complicated and inflexible. What if I want to truncate from the left, and pad to the right?

I propose making `alignStr` an implementation detail of `fmt`, deprecating it in favor of the following combinators:
```nim
truncateLeft(string, Natural): string
truncateRight(string, Natural): string
padLeft(string, Natural, string): string
padRight(string, Natural, string): string
```
e.g
```nim
alignStr(s, -2, trunc=true) == s.truncLeft(2).padLeft(2)
```